### ISSUE TITLE
v5.1.0 not found

### ISSUE THREAD

**Issue by mike-carey** [fastly/homebrew-tap#6]
It looks like the v5.1.0 release was potentially deleted or never published and now the formula is failing.

```
==> Fetching fastly/tap/fastly
==> Downloading https://github.com/fastly/cli/releases/download/v5.1.0/fastly_v5.1.0_linux-amd64.tar.gz
curl: (22) The requested URL returned error: 404
Error: fastly: Failed to download resource "fastly"
Download failed: https://github.com/fastly/cli/releases/download/v5.1.0/fastly_v5.1.0_linux-amd64.tar.gz
Installing fastly/tap/fastly has failed!
Homebrew Bundle failed! 1 Brewfile dependency failed to install.
Error: Process completed with exit code 1.
```

**Comment by mike-carey** [comment]
Quick update: looks like goreleaser failed and the release was not published: https://github.com/fastly/cli/actions/runs/4026844817/jobs/6922043350

**Comment by Integralist** [comment]
Hi @mike-carey 

Thanks for opening this issue and apologies for any inconvenience this caused.

There was a failure in the Fastly CLI's CI that was related to an issue with publishing to Arch Linux, which blocked us from completing the release process.

I managed to get the release published this morning:
https://github.com/fastly/cli/releases/tag/v5.1.0

But then discovered that Homebrew didn't update its SHASUM hashes when the CLI release was finally published (I think what happened was because the original CI error happened _after_ Homebrew was updated, re-starting the CI skipped the Homebrew section as there was already a 5.1.0 version set there).

If you run `brew upgrade fastly/tap/fastly` this should hopefully now retrieve the [updated formula](https://github.com/fastly/homebrew-tap/commit/52e63d705b09b0aa3ecebf06065fe8030439adb2) which will contain the SHASUM hashes updated to align with the [GitHub release assets file](https://github.com/fastly/cli/releases/download/v5.1.0/fastly_v5.1.0_SHA256SUMS).

I'll close this issue for now but if there are still problems then just let me know.

Thanks!
